cmd/devp2p/crawl.go

```diff
@@ -113,12 +140,17 @@ func (c *crawler) runIterator(done chan<- enode.Iterator, it enode.Iterator) {
 	}
 }
 
-func (c *crawler) updateNode(n *enode.Node) {
+// updateNode updates the info about the given node, and returns a status
+// about what changed
+// 0: node skipped
```

Comment:
`nodeRemoved = iota` comment seems a bit stale

Reply:
Yeah I'm just now removing that :)